### ISSUE TITLE
Add section to configure auto-recovery/backup frequency and options?

### ISSUE THREAD

**Issue by luzpaz** [howie-j/FreeCAD-Wizard#7]
![Screenshot_20220821_215912](https://user-images.githubusercontent.com/4140247/185824291-3f336aa0-b599-404c-9e60-ebd4c1db2532.png)

ref: https://wiki.freecad.org/Preferences_Editor#Document

Justification for this is to educate the user about auto-recovery/backup options in FreeCAD.

**Comment by howie-j** [comment]
IMHO this is a bit to detailed to be presented to new users, would it not be better to leave it as a sensible default and let users configure this themselves when needed?

**Comment by luzpaz** [comment]
fair enough. Maybe it can be a collapsed option that users can choose to address or not?
